build/lib/release.sh

```diff
@@ -478,12 +478,6 @@ EOF
   mkdir -p "${release_stage}/third_party"
   cp -R "${KUBE_ROOT}/third_party/htpasswd" "${release_stage}/third_party/htpasswd"
 
-  # Include only federation/cluster, federation/manifests and federation/deploy
-  mkdir "${release_stage}/federation"
-  cp -R "${KUBE_ROOT}/federation/cluster" "${release_stage}/federation/"
```

Comment:
Probably this directory `federation/cluster` is required. since federation-(up,down).sh are in the directory and our current test setup uses them.